### DISCUSSION TITLE
what siamrpn_r50_l234_dwxcorr_otb mean?

### DISCUSSION THREAD

**Discussion by Chenzixi1** [STVIR/pysot#495] [Q&A] [answered]
Does 'otb' in 'siamrpn_r50_l234_dwxcorr_otb' means you train with OTB2015 dataset? 

or? why the result is different from 'siamrpn_r50_l234_dwxcorr'?

**Comment by lb1100** [comment, marked as answer]
OTB is test dataset. Not for training.
